atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -69,15 +69,18 @@ def get_koji_upload_dir(workflow):
     return workspace[WORKSPACE_KEY_UPLOAD_DIR]
 
 
-def override_build_kwarg(workflow, k, v):
+def override_build_kwarg(workflow, k, v, platform=None):
```

Comment:
you can use default here directly 
platform='None'

Reply:
ahhh ok so None type is for all of em, got it :)

Reply:
Btw, this is ok because `None` is hashable to it can be used a dict key.